.github/workflows/publish_remote_config.yml

```
@@ -0,0 +1,23 @@
+name: Publish Remote Config
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    if: env.KILN_CONFIG_REPO == 'true'
```

actionlint (1.7.7)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)